templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,5 @@
+<%- dirnames = get_directories.map {|d| d.path } -%>
+
+<%- dirnames.each do |dir| -%>
+  Copy-Item <%= "#{dir.sub("/cygdrive/c", "C:/ProgramData/chocolatey/lib/#{project.name}")}/*" %> <%= dir.sub('/cygdirve/c', 'C:') %>
```

Comment:
What typo? I want to copy files around to get everything in the same directory. That way we can have all the libraries and binaries in predictable locations